airflow/www/views.py

```diff
@@ -2782,6 +2782,9 @@ def grid(self, dag_id: str, session: Session = NEW_SESSION):
         wwwutils.check_import_errors(dag.fileloc, session)
         wwwutils.check_dag_warnings(dag.dag_id, session)
 
+        included_events_raw = conf.get("webserver", "audit_view_included_events", fallback="")
+        excluded_events_raw = conf.get("webserver", "audit_view_excluded_events", fallback="")
```

Comment:
I'm a little confused, maybe you can explain it to me.

Deployment managers set these 2 to control what is shown in this UI view. I'm not sure we want to allow bypassing these configuration values, even if audit log access requires a high level of permissions.

Does this new feature honor this and just allow filtering based on the list of events the user would already see, or does it allow the user to make different choices than what the deployment manager configured?

Reply:
Okay, I pulled this down and I better understand what you're going for now. Noticed some issues though, going to elaborate in another comment.